### PR TITLE
chore: fixed list shuffles on RandomAppList 

### PR DESCRIPTION
**Pull request by developerfred**
## Description
Changing the logic of random numbers for correction or random reproduction
#1040

## Screenshots (if appropriate):
https://share.getcloudapp.com/2Nu5prrO

### CODE REVIEW

**Comment by developerfred** [comment]
@samajammin I believe I discovered the problem, we need to use hooks. Every time the component is shuffled the Intl  loads. Putting the component to be the last one to load using userEffects. makes the component work normally.

File: src/components/RandomAppList.js [hunk 1]
```diff
@@ -40,15 +40,22 @@ const appList = [
   },
 ]
 
-const RandomAppList = () => {
-  const list = appList.map((item) => {
-    item.randomNumber = Math.random()
-    return item
-  })
-  list.sort((a, b) => a.randomNumber - b.randomNumber)
+const RandomAppList = () =>
```
Comment (samajammin) [inline src/components/RandomAppList.js:43]: Missing brackets here which is breaking the build:
```
const RandomAppList = () => {
```

File: src/components/RandomAppList.js [hunk 2]
```diff
@@ -40,15 +40,22 @@ const appList = [
...
+const [setAppRandomList, appRandomList] = useState({ dapps: []});
```
Comment (samajammin) [inline src/components/RandomAppList.js:44]: The set state function should be the second item in the array.

Comment (samajammin) [reply src/components/RandomAppList.js:44]: I also don't think we need an object here (`useState({ dapps: []})`), I think we can just use an array `useState([])`

**Comment by developerfred** [comment]
Nice! Thanks @samajammin

**Comment by samajammin** [comment]
@all-contributors please add @developerfred for code.

**Pull request merged**